media-video/mkvtoolnix/mkvtoolnix-74.0.0.recipe

```diff
@@ -0,0 +1,141 @@
+SUMMARY="A Matroska file creator and editor"
+DESCRIPTION="MKVToolNix is a set of tools to create, alter and inspect \
+Matroska files. Matroska is a versatile media format which can contain \
+video, audio and subtitles, as well as chapter lists."
+HOMEPAGE="https://mkvtoolnix.download"
+COPYRIGHT="2003-2023 Moritz Bunkus"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://mkvtoolnix.download/sources/mkvtoolnix-$portVersion.tar.xz"
+SOURCE_DIR="mkvtoolnix-$portVersion"
+CHECKSUM_SHA256="b7dc8be6cb9170030de06b983f256ff3cb21efa3209f0efe8e079656fa3d5643"
+PATCHES="mkvtoolnix-$portVersion.patchset"
+ADDITIONAL_FILES="mkvtoolnix-gui.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mkvtoolnix$secondaryArchSuffix = $portVersion
+	app:MKVToolNixGUI$secondaryArchSuffix = $portVersion
+	cmd:mkvtoolnix_gui$secondaryArchSuffix = $portVersion
+	cmd:mkvmerge$secondaryArchSuffix = $portVersion
+	cmd:mkvextract$secondaryArchSuffix = $portVersion
+	cmd:mkvinfo$secondaryArchSuffix = $portVersion
+	cmd:mkvpropedit$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	docbook_xsl_stylesheets
+	libebml$secondaryArchSuffix >= 1.4.4
+	libmatroska$secondaryArchSuffix >= 1.7.1
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix >= 1.69.0
+	lib:libcmark$secondaryArchSuffix
+	lib:libdvdread$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
+	lib:libfmt$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libgmpxx$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libpugixml$secondaryArchSuffix
+	lib:libQt6Concurrent$secondaryArchSuffix
+	lib:libQt6Core$secondaryArchSuffix >= 6.1.0
```

Comment:
Versioning not needed here:
```diff
-	lib:libQt6Core$secondaryArchSuffix >= 6.1.0
+	lib:libQt6Core$secondaryArchSuffix
```
(this is because it's being taken care of with the devel packages)